### PR TITLE
Move discover message and add style

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #15507 .

### Summary of Changes
Move message to above the list to be more consistent and add standard styling

### After PR
screenshotr11-00-50

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :red_circle: unsuccessfully on 3ec3f849c0416107a7bea72356874a0679f41f81<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15528">issues.joomla.org/tracker/joomla-cms/15528</a>.</sub>

File: administrator/components/com_installer/views/discover/tmpl/default.php [hunk 1]
```diff
@@ -39,6 +39,10 @@
 				<?php echo JText::_('JGLOBAL_NO_MATCHING_RESULTS'); ?>
 			</div>
 			<?php else : ?>
+			<div class="alert alert-no-items alert-info">
```
Comment (Quy) [inline administrator/components/com_installer/views/discover/tmpl/default.php]: Remove `alert-no-items`?

Comment (brianteeman) [inline administrator/components/com_installer/views/discover/tmpl/default.php]: It is there as it is the existing class that gives a margin-top

**Comment by infograf768** [comment]
The way this patch is done, the message would show only when there are already extensions to discover in the list and not permanently as it was designed to be.

**Comment by infograf768** [comment]
imho
```
			<div class="alert alert-no-items alert-info">
				<?php echo JText::_('COM_INSTALLER_MSG_DISCOVER_DESCRIPTION'); ?>
			</div>
```
should go before
`<?php if (empty($this->items)) : ?>`

**Comment by brianteeman** [comment]
@franz-wohlkoenig @infograf768 fixed

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 0071f4302a79b4ac276a8cdecd436bc91dc6aa13<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15528">issues.joomla.org/tracker/joomla-cms/15528</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 0071f4302a79b4ac276a8cdecd436bc91dc6aa13<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15528">issues.joomla.org/tracker/joomla-cms/15528</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**